components/TARGET_PSA/TARGET_TFM/COMPONENT_SPE/secure_fw/spm/spm_api.c

```diff
@@ -286,7 +290,6 @@ uint32_t tfm_spm_partition_get_flags(uint32_t partition_idx)
             partition_flags;
 }
 
-#ifndef TFM_PSA_API
```

Comment:
Please comment in our patch (in the code, not the commit description) why this check is removed.

Reply:
This change is not relevant anymore. Retained the code from TF-M.